examples/plot_intersectional_bias.py

```diff
@@ -0,0 +1,1327 @@
+# of racism and sexism**. For instance, people's skin color was taken as being intrinsically
+# related to specific characteristics - a stereotype which has promoted all sorts of inequalities
+# in today's society.
+#
+# A notion of populations that emphasizes their dynamic nature is helpful because it helps
+# understand the relationships between these different groups, like gender, class, and religion -
+# all social constructions that have meaning for us. It acknowledges that **we're all working
+# within these systems through various different lenses, and within different dynamics of power and
+# privilege**. The graphic below is a great representation of that - of how varied all of our
+# different experiences can be:
+#
+#   *Intersectionality is a lens through which you can see where power comes and collides, where it
+#   locks and intersects. It is the acknowledgement that everyone has their own unique experiences
+#   of discrimination and privilege.* --Kimberle Crenshaw
+#
+# .. image:: ../_static/images/imhc_intersectionality.jpg
+#    :width: 600
+#    :alt: In the graphic above, Sylvia Duckworth uses a Spirograph to illustrate the multitude of
+#          ways that social identities might intersect. The Spirograph is split into 12 overlapping
+#          circles, each numbered, connected to a specific social identity, and assigned a unique
+#          colour. To illustrate the intersections of the different social identities, where each
+#          circle intersects, a new shade of the original colour is visible (as would happen when
+#          mixing paint colours together). At a glance the graphic shows all colours of the rainbow
+#          in different shades.
+#          The 12 social identities listed are: race, ethnicity, gender identity,
+#          class, language, religion, ability, sexuality, mental health, age, education,
+#          and body size.
+#          A quote from Kimberlé Crenshaw appears beneath the spirograph that reads
+#          “Intersectionality
+#          is a lens through which you can see where power comes and collides, where it locks and
+#          intersects.
+#          It is the acknowledgement that everyone has their own unique experiences of
+#          discrimination and privilege.” "Intersectionality" by Sylvia Duckworth, licensed under a
+#          `CC-BY-NC-ND`<https://creativecommons.org/licenses/by-nc-nd/2.0/> license.
+#
+# Critical race theory
+# ====================
+#
+# Ain't I A Woman?
+# ~~~~~~~~~~~~~~~~
+#
+# Note that although this intersectional approach has only recently been applied to ML, it is not a
+# novel concept. Rather, it’s a well-established idea that stretches back nearly 200 years to the
+# Black feminist and the suffragette movement, when white women were fighting for the right to
+# vote, but Black women seemed to be left out of this call for reform. There were a range of Black
```

Comment:
Nitpick: remove the word "seemed", just state "Black women were left out of this call for reform"